### ISSUE TITLE
New flow tower model doesn't seem to be as useful at gauging flow

### ISSUE THREAD

**Issue by Turbine1991** [kartchnb/AutoTowersGenerator#53]
I recently printed a flow tower and noticed it uses a different model. I've inspected it thoroughly, but unfortunately each layer seemed to come out the same.

The old model used to give me very different results and always worked perfectly.

Perhaps I'm missing something, but this may need discussing.

**Comment by kartchnb** [comment]
You're probably not missing anything.  The flow tower is a problem right now and I'm currently reworking how my scripts modify the gcode.  My plugin *does* modify the gcode, but nobody is seeing any real change between layers.  I'm not sure if it's the model or the way I'm modifying the gcode.  I should have an improvement soon.

I changed models because the old one was just something I dreamed up and I started questioning whether it was really the right option.  Thank yo for the feedback - I'll consider switching back.

Thank you for the error report.  It's very much appreciated.
